.github/workflows/backup-db.yml

```diff
@@ -78,3 +88,41 @@ jobs:
         backup-file: ${{ env.BACKUP_FILE }}.sql
         db-server-name: ${{ inputs.db-server }}
         slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
+
+    - name: Disk cleanup
+      if: github.event_name == 'schedule
+      shell: bash
+      run: |
+        sudo rm -rf /usr/local/lib/android || true
+        sudo rm -rf /usr/share/dotnet || true
+        sudo rm -rf /opt/ghc || true
+
+    - name: Sanitise dump
+      if: github.event_name == 'schedule
+      run: |
+        createdb ${DATABASE_NAME} && gzip -d --to-stdout ${{ env.BACKUP_FILE }}.sql.gz | psql -d ${DATABASE_NAME}
+        psql -d ${DATABASE_NAME} -f db/scripts/sanitise.sql
+        pg_dump --encoding utf8 --compress=1 --clean --no-owner --if-exists -d ${DATABASE_NAME} -f att_backup_sanitised.sql.gz
+      env:
+        DATABASE_NAME: apply_manage_itt
+        PGUSER: postgres
+        PGPASSWORD: postgres
+        PGHOST: localhost
+        PGPORT: 5432
+
+    - name: Remove backup file
+      if: github.event_name == 'schedule'
+      shell: bash
+      run: |
+        rm ${{ env.BACKUP_FILE }}.sql.gz
```

Comment:
I would move this line into the previous step. After line 103, before line 105.

Reply:
Resolved with commit d1500125eb7c6e2432fbae46f8651a3d346431d5